lib/mayaUsd/ufe/private/Utils.cpp

```diff
@@ -154,9 +154,13 @@ void applyCommandRestriction(const UsdPrim& prim, const std::string& commandName
         // if we have more than 2 layers that contributes to the final composed prim
         if (layers.size() > 1) {
             std::string layerDisplayNames;
-            for (auto layer : layers) {
-                layerDisplayNames.append("[" + layer->GetDisplayName() + "]" + ",");
-            }
+
+            // skip the the first arc which is PcpArcTypeRoot
+            // we are interested in all the arcs after root
+            std::for_each(std::next(layers.begin()),layers.end(), [&](const auto& it) {
+                 layerDisplayNames.append("[" + it->GetDisplayName() + "]" + ",");
+            });
```

Comment:
This change only affects the error report message. Using Kitchen_1 as an example, here is what the message currently reads:

`Cannot rename [Kitchen_1] with definitions or opinions on other layers. Opinions exist in [Kitchen_set.usd],[Kitchen.usd],[Kitchen_payload.usd],[Kitchen.geom.usd].`

The first arc is Pcp.ArcTypeRoot which points at [Kitchen_set.usd]. However since the message currently mentions "other layers", the error report could cause a bit of confusion. Therefore just displaying other layers in the error report:

`Cannot rename [Kitchen_1] with definitions or opinions on other layers. Opinions exist in [Kitchen.usd],[Kitchen_payload.usd],[Kitchen.geom.usd]. `